packages/remark-jargon/src/index.js

```diff
@@ -1,4 +1,4 @@
-import visit from 'unist-util-visit'
+import {visit} from 'unist-util-visit'
```

Comment:
this broke after I ran `yarn reconfigure` because it's now using a new version that only has named exports